templates/taxi/driver_form.html

```diff
@@ -0,0 +1,14 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+{% block content %}
+  <h1>{{ object|yesno:"Update,Create" }} Driver</h1>
+  <form action="" method="post" novalidate>
+    {% csrf_token %}
+    {% if objects.pk %}
+      {% form.license_number|crispy }}
```

Comment:
you pass different forms in different views, so you don't need this if, and dont need to pull out `form.license_number` explicitly since DriverLicenseUpdateForm holds only license number field